security/android_vbmeta.ksy

```diff
@@ -0,0 +1,192 @@
+meta:
+  id: android_vbmeta
+  title: Android VBMeta
+  license: CC0-1.0
+  file-extension: img
+  endian: be
+
+doc: |
+  Binary format for header of the vbmeta image.
+  The vbmeta image consists of three blocks:
+  +-----------------------------------------+
+  | Header data - fixed size                |
+  +-----------------------------------------+
+  | Authentication data - variable size     |
+  +-----------------------------------------+
+  | Auxiliary data - variable size          |
+  +-----------------------------------------+
+  The "Header data" block is described by this struct and is always
+  |AVB_VBMETA_IMAGE_HEADER_SIZE| bytes long.
+  The "Authentication data" block is |authentication_data_block_size|
+  bytes long and contains the hash and signature used to authenticate
+  the vbmeta image. The type of the hash and signature is defined by
+  the |algorithm_type| field.
+  The "Auxiliary data" is |auxiliary_data_block_size| bytes long and
+  contains the auxiliary data including the public key used to make
+  the signature and descriptors.
+  The public key is at offset |public_key_offset| with size
+  |public_key_size| in this block.
+  The descriptors starts at |descriptors_offset| from the beginning
+  of the "Auxiliary Data" block and take up |descriptors_size|
+  bytes. Each descriptor is stored as a |AvbDescriptor| with tag and
+  number of bytes following. The number of descriptors can be
+  determined by walking this data until |descriptors_size| is
+  exhausted.
```

Comment:
This docblock from the original code is completely redundant. It merely duplicates the structure described in `seq` and individual `types`.